common/models/order.json

```diff
@@ -73,6 +77,12 @@
       "principalId": "$owner",
       "permission": "ALLOW",
       "property": "__get__tickets"
+    },
+    {
+      "principalType": "ROLE",
+      "principalId": "$everyone",
```

Comment:
This should be the `$authenticated` role (or something similar) so that a user must be logged in or a 403 is thrown.